la_funding_analysis/pipeline/cleaning.py

```diff
@@ -0,0 +1,227 @@
+# File: pipeline/cleaning.py
+"""Functions to clean datasets.
```

Comment:
perhaps nice to add slightly more detail here i.e. examples of how to use the functions + you've called all your cleaning functions 'get_*' which seems more relevant for getters - maybe something more detailed i.e. get_clean_*?

Reply:
I've added some more detail here and now all "getting and cleaning" functions should be called get_clean_...